apps/jobs/views.py

```diff
@@ -528,8 +528,18 @@ def get_submission_by_pk(request, submission_id):
                 description='Submission error file content'
             ),
             'result': openapi.Schema(
-                type=openapi.TYPE_STRING,
-                description='Submission results'
+                type=openapi.TYPE_OBJECT,
+                description='Submission results (The keys in this string must correspond to the leaderboard schema format. There can be n number of keys.)',
```

Comment:
Replace `description='Submission results (The keys in this string must correspond to the leaderboard schema format. There can be n number of keys.)',` with 
`description='Submission results in dict format (Keys in this dict must correspond to the metrics in the leaderboard of challenge phase split. API will throw an error if any metric is missing)',`

Reply:
Also, just to confirm: Is it supposed to be array of dicts or just dict?

Reply:
It is just dict.